src/status_im/ui/screens/db.cljs

```diff
@@ -18,11 +18,14 @@
             status-im.ui.screens.browser.db
             status-im.ui.screens.add-new.db))
 
-(def transaction-send-default
-  {:symbol    :ETH
-   :gas       ethereum/default-transaction-gas
+(defn gas-default []
+  {:gas       ethereum/default-transaction-gas
    :gas-price ethereum/default-gas-price})
 
+(defn transaction-send-default []
+  (merge (gas-default)
+         {:symbol    :ETH}))
```

Comment:
why is it a function? seems like it can be easily transformed to variable.

Reply:
I did this change to prepare for the move to a computed gas price (#3241)

Reply:
could you leave a comment?